### PR TITLE
Update gas station article

### PR DESCRIPTION
**Pull request by bucanero**

### CODE REVIEW

**Comment by bucanero** [comment]
@ravigautham , we need to update the "contact us" link about automated settlement before we merge this PR

Please let me know if we have a link or form ready for this action.

**Comment by ravigautham** [comment]
@bucanero Here's the link to the google form https://forms.gle/7z9nKVd4VH3qxbny6

File: docs/2.build/1.chain-abstraction/multichain-gas-relayer/gas-station.md [hunk 1]
```diff
@@ -66,6 +70,17 @@ Manual Settlement involves several steps:
...
    - A swap from the bridged token to the native gas token before sending to the paymaster accounts is necessary if the token that was bridged was not the foreign chain native gas token
 
+### Automated settlement
+
+Automated settlement is available for select partners based on their cross-chain transaction volume.
+When using automated settlement, a Market Maker facilitates the settlement of assets between a partner’s gas station contract and their paymaster accounts on destination chains.
+
+:::info Contact us
+
+If you're interested in using the automated settlement service, please [contact us](#) by filling out [this form](#).
```
Comment (bucanero) [inline docs/2.build/1.chain-abstraction/multichain-gas-relayer/gas-station.md:80]:
```suggestion
If you're interested in using the automated settlement service, please [contact us](https://forms.gle/7z9nKVd4VH3qxbny6) by filling out [this form](https://forms.gle/7z9nKVd4VH3qxbny6).
```

**Comment by bucanero** [comment]
> @bucanero Here's the link to the google form https://forms.gle/7z9nKVd4VH3qxbny6

Thanks! PR updated with the link ✅

**Review by ravigautham — APPROVED** [review]

**Pull request merged**